cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -440,6 +462,28 @@ CREATE INDEX ...
 a|
 The default index type is changed from B-tree to range index.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW INDEXES
+----
+a|
+The new column `owningConstraint` was added and will be returned by default from now on.
+It will list the name of the constraint that the index is associated with or `null`, in case it is not associated with any constraint.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW CONSTRAINTS
+----
+a|
+The new column `ownedIndex` was added and will be returned by default from now on.
```

Comment:
Why this inconsistency?

```
ownedIndex
owningConstraint
```

`owned` vs `owning` ???

Reply:
I believe this has been answered on slack :)